app.py

```diff
@@ -83,7 +84,7 @@ def get_data_24h_in(byhour):
 
 app = dash.Dash()
 app.title = 'Weather || Astrool'
-image_filename = 'obj.png' # replace with your own image
+image_filename = 'logo.png' 
```

Comment:
This looks like the intended filename btw

Reply:
Oh thanks!